engine/settings/base.py

```diff
@@ -852,4 +852,7 @@ class BrokerTypes:
 ACKNOWLEDGE_REMINDER_TASK_EXPIRY_DAYS = os.environ.get("ACKNOWLEDGE_REMINDER_TASK_EXPIRY_DAYS", default=14)
 
 KAVENEGAR_API_KEY=os.environ.get("KAVENEGAR_API_KEY", None)
-KAVENEGAR_VERIFICATION_SMS_TEMPLATE=os.environ.get("KAVENEGAR_VERIFICATION_SMS_TEMPLATE", None)
\ No newline at end of file
+KAVENEGAR_VERIFICATION_SMS_TEMPLATE=os.environ.get("KAVENEGAR_VERIFICATION_SMS_TEMPLATE", None)
+
+
+TELEGRAM_API_ENDPOINT=os.environ.get("KAVENEGAR_VERIFICATION_SMS_TEMPLATE", "https:api.telegram.org")
```

Comment:
regardless that you entered default value wrong.

it's better to set this value just with domain without `https`. just `api.telegram.org`